games/p.yaml

```
@@ -486,6 +499,364 @@
   - 'https://raw.githubusercontent.com/pokemium/PokeTraveler/master/docs/demo3.png'
   - 'https://raw.githubusercontent.com/pokemium/PokeTraveler/master/docs/demo4.png'
 
+- name: Pop3-Scripts
+  originals:
+  - 'Populous: The Beginning'
+  type: tool
+  repo: 'https://github.com/MrKosjaK/Pop3-Scripts'
+  development: halted
+  langs:
+  - Lua
+  licenses:
+  - Custom
```

```suggestion
  - As-is
```
same here, "As-is" means no license